### PR TITLE
Check_Evaluation_Status

### PR DESCRIPTION
**Pull request by PormpimonPang**

### CODE REVIEW

File: lib/models/evaluation_model.dart [hunk 1]
```diff
@@ -53,15 +53,15 @@ class EvaluationModel {
...
     {
-      "formname": "แบบประเมินการประเมินแผล",
+      "formname": "แบบประเมินแผล",
       "selectedform": EvaluationFormTopic.surgicalIncisionHome,
     },
     {
       "formname": "แบบประเมินอาการแสดงที่ผิดปกติ",
       "selectedform": EvaluationFormTopic.abnormalSymptomHome,
     },
     {
```
Comment (Thaweesak-sai) [inline lib/models/evaluation_model.dart:63]: Should use camelCase

**Review by Thaweesak-sai — CHANGES_REQUESTED** [review]

File: lib/models/evaluation_model.dart (hunk 1 above)
Comment (PormpimonPang) [reply lib/models/evaluation_model.dart]: a6e4f48 ka

File: lib/services/calculation_service.dart [hunk 2]
```diff
@@ -23,13 +23,20 @@ class CalculationService extends ICalculationService {
...
     return bmi.toStringAsFixed(2);
   }
 
   String calculateBML(int oldWeight, int weight) {
+    oldWeight = oldWeight ?? 0;
+    weight = weight ?? 0;
+
     var bml = ((oldWeight - weight) / oldWeight) * 100;
+    if (bml.isNaN || bml.isInfinite) bml = 0;
     return bml.toStringAsFixed(2);
```
Comment (Thaweesak-sai) [inline lib/services/calculation_service.dart:40]: Then should fix the test as well

**Review by Thaweesak-sai — CHANGES_REQUESTED** [review]

File: lib/services/calculation_service.dart (hunk 2 above)
Comment (PormpimonPang) [reply lib/services/calculation_service.dart]: 6dfc8e9 Fixed ka

File: test/services_test/calculation_service_test.dart [hunk 3]
```diff
@@ -13,12 +13,59 @@ void main() {
...
+      expect(result, '2.86');
+      expect(result2, '-2.94');
+    });
+
+    test('Calculate BML should handle null value', () {
+      ICalculationService _calculationService = locator<ICalculationService>();
+      var result = _calculationService.calculateBWL(null, null);
+      expect(result, '0.00');
+    });
+  });
```
Comment (Thaweesak-sai) [inline test/services_test/calculation_service_test.dart:68]: Why we have 2 set of the same test here?

Comment (PormpimonPang) [reply test/services_test/calculation_service_test.dart:68]: @Thaweesak-sai  The first one for BMI(Body mass index) test but the second one for BWL(Body weight loss) test.

**Review by Thaweesak-sai — APPROVED** [review]

**Pull request merged**